app/travel_analytics/webapp/annotations/annotation.xml

```diff
@@ -0,0 +1,18 @@
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
```

Comment:
Why isn't that generated by us as part of `2edmx`?

Reply:
file seems to be unnecessary, not presetn in travel_processor -> removed

Reply:
@c-kobo @stefan0816 : this file seems to be unnecessary for a CAP app -> can the generation tool in vs code be changed so that this file isn't generated anymore?